PennMobile/src/main/java/com/pennapps/labs/pennmobile/MainActivity.kt

```diff
@@ -95,11 +95,18 @@ class MainActivity : AppCompatActivity() {
         // Show HomeFragment if logged in, otherwise show LoginFragment
         val pennKey = mSharedPrefs.getString(getString(R.string.pennkey), null)
         val guestMode = mSharedPrefs.getBoolean(getString(R.string.guest_mode), false)
+        var diningWidgetBroadCast = 0
```

Comment:
can you move this like 5 lines down to be with the other diningWidget stuff. Even better, you should put this into a new function.